### PR TITLE
Fix odd wording around solo gimbal cable connectors

### PR DESCRIPTION
**Pull request by hamishwillee**
Fix wording for solo gimbal bay connectors.

### CODE REVIEW

**Comment by hamishwillee** [comment]
This still needs minor clarification on the 5V supply - what does 5.38V output "mean". There are lots of other fixes needed too.

**Comment by hamishwillee** [comment]
@mrpollo Can we get this merged please.

File: book/hardware-gimbalbay.md [hunk 1]
```diff
@@ -1,54 +1,92 @@
...
-4 | GND (USB) | Brown
-5 | Gimbal Rx | White
-6 | Gimbal Tx | Green
-7 | USB D+ | Blue
-8 | USB D- | Violet
+We recommend that you use the *Solo Gimbal Beauty Plate* as the basis for your own connector plate, as it correctly implements the required mechanical interface points. The CAD file for the plate can be [downloaded from here](https://docs.google.com/a/3drobotics.com/uc?id=0B9l93ZUM5ooxT3lSeHhwZ3VvSmc&export=download) and a screenshot is shown below.
+
+![Solo Beauty Plate](images/solo_gimbal_bay_beauty_plate.png)
+
+The maximum payload of the system is 700g (for Camera, Gimbal and other accessories). Camera/Gimbal solutions that exceed 390g may not work with accessories that are designed to work with the 3DR Gimbal and GoPro.
```
Comment (peterbarker) [inline book/hardware-gimbalbay.md]: s/the system/Solo/
?

File: book/hardware-gimbalbay.md [hunk 2]
```diff
@@ -1,54 +1,92 @@
...
 
-## Hardware spec
 
 <aside class="todo">
```
Comment (peterbarker) [inline book/hardware-gimbalbay.md]: No longer todo

File: book/hardware-gimbalbay.md [hunk 3]
```diff
@@ -1,54 +1,92 @@
...
 
 <aside class="note">
-The continuous current draw is limited by the gimbal cable despite the actual supply being capable of 25W (5A). This supply is also shared with some other functions on the copter.
+Communication between the Solo co-processor and the gimbal is not yet available for 3rd party developers.
 </aside>
 
-### HDMI Mini 
+### HDMI Micro Cable
 
 The [HDMI](https://en.wikipedia.org/wiki/HDMI) Mini connection is responsible for transferring video from the camera on the Solo to the first person view in the 3DR app. The HDMI connection does not have the audio pins connected. The video feed supports up to 1080p resolution at 60 frames per second.
```
Comment (peterbarker) [inline book/hardware-gimbalbay.md]: This also needs correction to "Micro" (Type D) (rather than "Mini")

File: book/hardware-gimbalbay.md [hunk 4]
```diff
@@ -63,15 +101,21 @@ Supported resolutions:
...
+
 ## Software Interface
 
+<aside class="caution">
+The Solo software stack is not yet ready to integrate multiple types of gimbals. We are currently working on an extensible API to make it easier for third-party vendors to interface with different hardware.
+
+For development purposes we recommend using Pixhawk 1 and Copter 3.3 Ardupilot firmware.
+</aside>
+
 In order to have Solo control a gimbal, Ardupilot must know how to communicate with the gimbal. Fortunately, Ardupilot already has the ability to communicate with several types of pre-existing gimbal controllers including *SToRM32* and *Alexmos*. The custom Solo Gimbal protocol is not yet supported by any released versions of Ardupilot.
```
Comment (peterbarker) [inline book/hardware-gimbalbay.md]: ArduPilot (camel-case) is apparently the blessed form :-)

File: book/hardware-gimbalbay.md (hunk 1 above)
Comment (hamishwillee) [inline book/hardware-gimbalbay.md]: This will be fixed by next commit

File: book/hardware-gimbalbay.md (hunk 4 above)
Comment (hamishwillee) [inline book/hardware-gimbalbay.md]: Yes it is :-)

**Comment by hamishwillee** [comment]
@peterbarker Last comment fixes your comments (Thanks!). Provided tests pass this can be merged.

**Pull request merged**